src/verso-manual/VersoManual/Bibliography.lean

```diff
@@ -167,6 +186,13 @@ def Citable.bibHtml (go : Doc.Inline Genre.Manual → HtmlT Manual (ReaderT Exte
   | .arXiv p =>
     let authors ← andList <$> p.authors.mapM go
     return {{ {{authors}} s!", {p.year}. " {{ link {{"“" {{← go p.title}} "”"}} }} ". arXiv:" {{p.id}} }}
+  | .book p =>
+    let authors ← andList <$> p.authors.mapM go
+    return {{
+      {{authors}} s!", {p.year}. " {{ link {{"“" {{← go p.title}} "”"}} }} ". "
+      {{ p.series.map (λ (s, v) => {{ "Volume " <strong>s!"{v}"</strong> " of " /- TODO: insert series -/ ", "}}) |>.getD .empty }}
```

Comment:
I assume it's rare for a book reference to include only series or volume without the other, so I’ve treated them as a pair. However, I’m unsure how to represent this pairing properly in the citation formatting (manupulating `Html` or etc. is too complex).

Reply:
See comment on datatype